src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/jsonb/JsonbTransientAnnotationQuickFix.java

```diff
@@ -35,20 +36,15 @@ public class JsonbTransientAnnotationQuickFix extends RemoveMultipleAnnotations
     protected List<List<String>> getMultipleRemoveAnnotations(Project project, List<String> annotations) {
         List<List<String>> annotationsListsToRemove = new ArrayList<List<String>>();
 
-        if (annotations.contains(JsonbConstants.JSONB_TRANSIENT)) {
+        if (annotations.contains(JsonbConstants.JSONB_TRANSIENT_FQ_NAME)) {
             // Provide as one option: Remove JsonbTransient
             annotationsListsToRemove.add(Arrays.asList(JsonbConstants.JSONB_TRANSIENT_FQ_NAME));
         }
 
         // Provide as another option: Remove all other JsonbAnnotations
-        annotations.remove(JsonbConstants.JSONB_TRANSIENT);
+        annotations.remove(JsonbConstants.JSONB_TRANSIENT_FQ_NAME);
         if (annotations.size() > 0) {
-            // Convert the short annotation names to their fully qualified equivalents.
-            List<String> fqAnnotations = new ArrayList<>();
-            for (String annotation : annotations) {
-                fqAnnotations.addAll(getFQAnnotationNames(project, annotation));
-            }
-            annotationsListsToRemove.add(fqAnnotations);
+            annotationsListsToRemove.addAll(Collections.singleton(annotations));
```

Comment:
This seems to do the same thing as the old line of code but is less efficient and harder to understand. Why wrap the list in a collection?

Reply:
Done